Self-Consistent-Field/vcd.py

```diff
@@ -0,0 +1,785 @@
+psi_m2angstroms = 10**10                                                                                        # m -> Ang
+#
+# Mass conversions
+amu2kg = psi4.constants.get("Atomic mass constant")                                                             # amu -> kg
+psi_au2amu = psi4.constants.au2amu                                                                              # m_e -> amu
+#
+# Dipole moment type conversions
+psi_dipmom_au2debye = psi4.constants.dipmom_au2debye                                                            # e a0 -> D (C m)
+psi_dipmom_au2si = psi4.constants.dipmom_au2si                                                                  # e a0 -> C m
+conv_dip_grad_au2DAamu = (psi_dipmom_au2debye / psi_bohr2angstroms) * (1 / np.sqrt(psi_au2amu))                 # (e a0 / a0)/(m_e^(1/2)) -> (D/A)/(amu^(1/2))
+conv_cgs = 6.46047502185 * (10**(-36))                                                                          # (e a0)^2 -> (esu * cm)^2
+#
+# IR frequency conversions
+omega2nu = 1./(psi_c*2*np.pi)                                                                                   # omega (s^-1)  -> nu (cm^-1)
+psi4_hartree2wavenumbers = psi4.constants.hartree2wavenumbers                                                   # Eh -> cm^-1
+conv_ir_au2DAamu = conv_dip_grad_au2DAamu ** 2                                                                  # (e a0 / a0)^2/m_e -> (D/A)^2/amu
+conv_freq_au2wavenumbers = np.sqrt((1 / psi_au2amu) * hartree2joule / (sqbohr2sqmeter * amu2kg)) * omega2nu     # (Eh / (bohr^2 m_e)) -> cm^-1
+conv_freq_wavenumbers2hartree = 1 / psi4_hartree2wavenumbers                                                    # cm^-1 -> Eh
+
+# Convert IR intensities from (D/A)^2/amu to km/mol:
+#
+# Conversion factor for taking IR intensities from ((D/A)^2 / amu) to a.u. ((e a0 / a0)^2 / me):
+conv_kmmol = (1 / conv_dip_grad_au2DAamu) ** 2
+# Multiply by (1 / (4 pi eps_0)) in a.u. (1 / (e^2 / a0 Eh)) to get to units of (Eh a0 / me)
+# Multiply by (Na pi / 3 c^2) in a.u. (Na = mol^-1; c = Eh / alpha^2 me) to get to units of a0/mol
+conv_kmmol *= psi_na * np.pi * (1 / 3) * psi_alpha**2
+# Convert to units of km/mol
+conv_kmmol *= psi_bohr2m * (1 / 1000)
+#
+conv_ir_DAamu2kmmol = conv_kmmol                                                                                # (D/A)^2/amu -> km/mol
+
+
+# Specify Molecule
+mol = psi4.geometry("""
+    O       0.0000000000        1.3192641900       -0.0952542913
+    O      -0.0000000000       -1.3192641900       -0.0952542913
+    H       1.6464858700        1.6841036400        0.7620343300
+    H      -1.6464858700       -1.6841036400        0.7620343300
+    symmetry c1
+    units bohr
+    noreorient
+    no_com
+""")
+
+psi4.core.set_active_molecule(mol)
+
+geom = psi4.core.Matrix.to_array(mol.geometry())
+
+# Set Psi4 Options
+options = {'BASIS':'STO-3G_dalton',
```

Comment:
Is this basis set in Psi4 now?

Reply:
No, I defined a custom basis set here that I'm calling `STO-3G_dalton` where I've adjusted the coefficients for H and O to match DALTON's STO-3G basis set definition. Do you think this is appropriate? If not, I can specify a custom basis set in DALTON to match Psi4/BSE and re-run the DALTON code. Just let me know if you'd prefer that instead.

Reply:
I had scrolled through the definition of the custom basis :facepalm: This is better than re-running DALTON with the "correct" basis, I think.